src/consent/set.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import * as r from 'ramda';
+
+import {
+  aliasesToIds,
+} from './map';
+import get from './get';
+
+type Categories = {
+  [id: string]: boolean,
+};
+
+const set = (
+  mappings: Object,
+  consent: string,
+  categories: Categories,
+) => {
+  const idCategories = aliasesToIds(categories, mappings);
+
+  const formatCategories = r.pipe(
+    aliasesToIds(r.__, mappings),
+    r.mapObjIndexed(
+      (category, id) => r.pipe(
+        r.prop(id),
+        r.ifElse(
+          r.equals(true),
+          r.always(0),
+          r.always(1),
+        ),
+        (value) => `${id}:${value}`,
+      )(idCategories),
+    ),
+    r.values,
+    r.join('|'),
+  );
+
+  return r.pipe(
+    () => get(mappings, consent),
```

Comment:
Pipe takes a varidic number of args, so you can just pass `mappings, consent` to `pipe` where the first operation is `get`